packages/devtools_app/lib/src/timeline/html_timeline_screen.dart

```diff
@@ -387,9 +387,7 @@ class HtmlTimelineScreen extends HtmlScreen {
   }
 
   void _configureSplitter({List<int> sizes}) {
-    if (sizes != null) {
-      assert(sizes.length == 2);
-    }
+    assert(sizes?.length ?? 2 == 2);
```

Comment:
nit: probably simpler to just write
assert(size == null || sizes.length == 2)

Reply:
that is simpler. Done.